style.css

```diff
@@ -1,7 +1,7 @@
 :root {
   --font-medium: 1.4rem;
   --primary-color: white;
-  --secondary-color: black;
+  --secondary-color: royalblue;
```

Comment:
try to use HEX or rgba(cause different browsers interpret same color differently). Rest everything looks good.
All the best.

Reply:
sure will change it

Reply:
done